aurorawatch-pipeline/README.md

```diff
@@ -36,7 +36,7 @@ aws configure
 This will require sensitive information to be entered through the command line.
 
 5. Add to your `.env` file the following:
-    - REGION - region of your ECR repository
+    - AWS_REGION - region of your ECR repository
```

Comment:
Good change to keep in naming conventions with the terraform

Reply:
Good change to keep in naming conventions with the terraform

Reply:
AWS_REGION is the tfvars env variable not REGION, LGTM.